resources/log4j2.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+  <Properties>
+    <Property name="SYSLOG_PORT_514_UDP_ADDR">localhost</Property>
+    <Property name="SYSLOG_PORT_514_UDP_PORT">514</Property>
+  </Properties>
+  <Appenders>
+    <Syslog name="Syslog" format="RFC5424" appName="BatchAddress"
```

Comment:
This handles Loggly, but we're talking about moving to Papertrail; is the setup consistent between the two?